src/events.ts

```diff
@@ -65,6 +66,30 @@ async function sendEvent(event, to) {
   return res.json();
 }
 
+export async function sendEventToWebhook(event) {
+  if (!webhookURL) return;
+  // TODO: handle errors and retry, maybe save in temporary table
+  try {
+    // TODO: This secret is temporary key, replace this with event URL once events code is removed
+    const secret = sha256(`${webhookURL}/api/webhook?cb=1${serviceEventsSalt}`);
+    const res = await fetch(webhookURL + '/api/event', {
```

Comment:
We should always use template literals
```suggestion
    const res = await fetch(`${webhookURL}/api/event`, {
```